--- nsca-2.9.1/./src/nsca.c~	2012-12-11 17:19:37.000000000 +0200
+++ nsca-2.9.1/./src/nsca.c	2012-12-11 17:21:43.768863034 +0200
@@ -1263,7 +1263,8 @@
         fclose(checkresult_file_fp);
         /* create and close ok file */
         asprintf(&checkresult_ok_file,"%s.ok",checkresult_file);
-        syslog(LOG_DEBUG,"checkresult completion file '%s' open.",checkresult_ok_file);
+        if(debug==TRUE)
+			syslog(LOG_DEBUG,"checkresult completion file '%s' open.",checkresult_ok_file);
         checkresult_ok_file_fp = fopen(checkresult_ok_file,"w");
         fclose(checkresult_ok_file_fp);
         /* reset umask */
