--- nsca-2.6/sample-config/nsca.cfg.in	2006-07-07 16:38:26.632832206 +0300
+++ nsca-2.9.1/sample-config/nsca.cfg.in	2012-06-26 01:40:43.486716471 +0300
@@ -15,7 +15,7 @@
 # directly into the checkresult directory.  Requires Nagios 3+
 # For best results, mount dir on ramdisk.
 
-#check_result_path=/usr/local/nagios/var/checkresults
+check_result_path=/var/spool/nagios/checkresults
 
 
 # PID FILE
@@ -61,7 +61,7 @@
 # NOTE: if you specify this option, the command file will be opened
 #       relative to this directory.
 
-#nsca_chroot=/var/run/nagios/rw
+#nsca_chroot=@localstatedir@/rw
 
 
 
